### PR TITLE
[Live-Site] Added event code model

### LINKED ISSUE #52: [Live-Site] Event Code Model
## Description

The event code model is required for the live screen share feature in live site to store the event code generated for mavens to join. 

## Parent Story
https://github.com/Real-Dev-Squad/website-www/issues/343

## Detailed requirements
[Requirement document](https://precodes.notion.site/Requirement-document-for-live-screen-share-feature-e8c0c1eea04947beb44ddb60e609aaec)

## Model Structure

```
{
	id: "",
	code: "",
	created_at : "",
	updated_at : ""
}
```

### PR DESCRIPTION
**Pull request by prerana1821**
## Description

Event code model is required for the live screen share feature to store the event code generated for mavens to join.

## Fixes

#52

### CODE REVIEW

**Review by Pratiyushkumar — DISMISSED** [review]

File: event-code/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Event Code
+
+```
+{
+	id: string,
+	code: string,
+	created_at: <timestamp>,
+	updated_at: <timestamp>
+}
```
Comment (SanketDhabarde) [inline event-code/README.md:9]: Can you please check once I think we used to add `role` as well

Comment (prerana1821) [reply event-code/README.md:10]: Done

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by SanketDhabarde — APPROVED** [review]
LGTM

**Review by Pratiyushkumar — APPROVED** [review]

**Pull request merged**